### PR TITLE
[docker-sonic-vs/Dockerfile.j2]: Install new YANG PKGs in VS docker.

### PR DESCRIPTION
**Pull request by praveen-li**
Changes:
. Install new YANG PKGs in VS docker.
. Install fixed version on other PY PKGs.

Signed-off-by: Praveen Chaudhary pchaudhary@linkedin.com

**- What I did**
 Install new YANG PKGs in VS docker.

**- How I did it**
Added in packages in 
$(DOCKER_SONIC_VS)_PYTHON_WHEELS += $(SONIC_YANG_MODELS_PY3)
$(DOCKER_SONIC_VS)_PYTHON_WHEELS += $(SONIC_YANG_MGMT_PY)

**- How to verify it**
At http://jinja.quantprogramming.com/
```
{% if docker_sonic_vs_whls.strip() %}
# copy all whl PKGs first,
{% for whl in docker_sonic_vs_whls.split(' ') -%}
COPY python-wheels/{{ whl }}  python-wheels/
{% endfor %}

# install PKGs after copying all PKGs to avoid dependency failure
# use py3 to find python3 package, which is forced by wheel as of now
{%- for whl in docker_sonic_vs_whls.split(' ') %}
{%- if 'py3' in whl %}
RUN pip3 install python-wheels/{{ whl }}
{% else -%}
RUN pip install python-wheels/{{ whl }}
{%- endif %}
{%- endfor %}
{% endif %}
```
Input:

Builds DEP PKGs at same time with VS docker due to dependencies:
```
pchaudha@server05:/home/pchaudha/srcCode/dpb_repo$ ls -l target/docker-sonic-vs.gz.log
-rw-r--r-- 1 pchaudha pchaudha 35345 Apr 21 15:37 target/docker-sonic-vs.gz.log
pchaudha@server05:/home/pchaudha/srcCode/dpb_repo$ ls -l target/python-wheels/sonic_yang_m*
-rw-r--r-- 1 pchaudha pchaudha 13431 Apr 21 15:34 target/python-wheels/sonic_yang_mgmt-1.0-py2-none-any.whl
-rw-r--r-- 1 pchaudha pchaudha  4036 Apr 21 15:34 target/python-wheels/sonic_yang_mgmt-1.0-py2-none-any.whl.log
-rw-r--r-- 1 pchaudha pchaudha 13219 Apr 21 15:34 target/python-wheels/sonic_yang_models-1.0-py3-none-any.whl
-rw-r--r-- 1 pchaudha pchaudha   662 Apr 21 15:34 target/python-wheels/sonic_yang_models-1.0-py3-none-any.whl-install.log
-rw-r--r-- 1 pchaudha pchaudha 15225 Apr 21 15:34 target/python-wheels/sonic_yang_models-1.0-py3-none-any.whl.log
```

Installed
```
1338  ---> ee4aeae03107
1339 Step 68/95 : COPY python-wheels/sonic_yang_models-1.0-py3-none-any.whl  python-wheels/
1340  ---> 7c00d50ab8df
1341 Step 69/95 : COPY python-wheels/sonic_yang_mgmt-1.0-py2-none-any.whl  python-wheels/
1342  ---> e4fe41dd566c
1343 Step 70/95 : RUN pip3 install python-wheels/sonic_yang_models-1.0-py3-none-any.whl
1344  ---> Running in f80944aa9043
1345 Processing /python-wheels/sonic_yang_models-1.0-py3-none-any.whl
1346 Installing collected packages: sonic-yang-models
1347 Successfully installed sonic-yang-models-1.0
1348 Removing intermediate container f80944aa9043
1349  ---> 485c67df7190
1350 Step 71/95 : RUN pip install python-wheels/sonic_yang_mgmt-1.0-py2-none-any.whl
1351  ---> Running in afd023adb15a
1352 ^[[91mDEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer ma     intained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://     pip.pypa.io/en/latest/development/release-process/#python-2-support
1353 ^[[0mProcessing /python-wheels/sonic_yang_mgmt-1.0-py2-none-any.whl
1354 Installing collected packages: sonic-yang-mgmt
1355 Successfully installed sonic-yang-mgmt-1.0
1356 Removing intermediate container afd023adb15a
```
Jenkins Test Results:
http://172.21.47.20:8080/job/DPB%20smoke%20test/53/console

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by praveen-li** [comment]
retest vsimage please

File: build_debian.sh [hunk 1]
```diff
@@ -445,6 +445,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'docker
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install gcc libpython2.7-dev
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'netifaces==0.10.7'
 
+# Get package to support Dynamic Port Breakout
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install xmltodict==0.12.0
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install jsondiff==1.2.0
```
Comment (lguohan) [inline build_debian.sh:450]: this does not seem to be related to vs docker.

Comment (praveen-li) [reply build_debian.sh:450]: Yeah It is not, let me start another PR for it, since these 2 PKGs are needed on Sonic Switch to run Sonic_utilities/config/config_mgmt.py Library. Thx

Comment (praveen-li) [reply build_debian.sh]: Fixed.

**Comment by praveen-li** [comment]
retest broadcom please
retest vsimage please

**Comment by praveen-li** [comment]
retest vsimage please

**Comment by praveen-li** [comment]
retest broadcom please
